v1.1.1/js/Sidebar.js

```diff
@@ -210,6 +227,11 @@ function Sidebar(loopy){
 			"<span class='mini_button' onclick='publish(\"debug/toggle\")'>toggle debug values</span> <br><br>"+
 			"<span class='mini_button' onclick='publish(\"model/resetZoom\")'>reset zoom</span> <br><br>"+
 			"<span class='mini_button' onclick='publish(\"graph/toggleVisible\")'>toggle Graph visibility</span> <br><br>"+
+			"Color Palette: <br>" +
+			"<span class='mini_button' onclick='publish(\"node/changeColorRainbow\")'>Rainbow</span> "+
+			"<span class='mini_button' onclick='publish(\"node/changeColorTol\")',>Tol</span> "+
+			"<span class='mini_button' onclick='publish(\"node/changeColorWong\")'>Wong</span><br><br>"+
+			// "<span class='mini_button' onclick='publish(\"node/changeColorBW\")'>Monochrome</span><br><br>"+
```

Comment:
Is there still some issue with the monochrome version, or was this just accidentally left commented out? Ideally we'd either fix the issue if possible, or if we're abandoning the monochrome palette then we would just delete that line rather than comment it out.

Reply:
Fixed: Removed. Was gonna ask if you thought monochrome was worth salvaging or not - has to do with visibility of text over filled node / empty node when color is black / close to black